src/libs/ajax/User.ts

```diff
@@ -76,15 +76,23 @@ export interface SetTerraUserProfileRequest {
 
 export const makeSetUserProfileRequest = (terraUserProfile: TerraUserProfile): SetTerraUserProfileRequest => {
   return {
-    firstName: terraUserProfile.firstName ?? 'N/A',
-    lastName: terraUserProfile.lastName ?? 'N/A',
-    title: terraUserProfile.title ?? 'N/A',
-    institute: terraUserProfile.institute ?? 'N/A',
+    // first name and last name are REQUIRED for this request, but should not ever be "N/A"
+    // TODO: determine if this request should fail if firstname or lastname is undefined
+    firstName: terraUserProfile.firstName! ?? 'N/A',
+    lastName: terraUserProfile.lastName! ?? 'N/A',
+
+    // title and institute are REQUIRED for this request, but they do not necessarily
+    // get set during registration
+    title: !_.isEmpty(terraUserProfile.title) ? terraUserProfile.title : 'N/A',
+    institute: !_.isEmpty(terraUserProfile.institute) ? terraUserProfile.institute : 'N/A',
```

Comment:
Why are some fields checked with the nullish coalescing operator and some with `_.isEmpty`?

Reply:
These fields are both required and cannot be `''`. When I had manual tested this with the `??` operator and neither of these fields were filled out, it evaluated to the left side and failed the request.

Reply:
That makes sense. But doesn't that same issue exist for `programLocationCity`? Should those also be checked using `_.isEmpty`?

Reply:
Those are not required fields, so I dont think they need to be a nonEmpty value, but let me confirm with a manual test.

Reply:
![image](https://github.com/DataBiosphere/terra-ui/assets/56648914/6856e596-4670-4430-a01a-a150710d1beb)
Good catch 👍

Reply:
There's a comment that says program location fields are required.

Reply:
That is actually also in prod now, so I am raising this to my team and see what we need to do

Reply:
Ok. Maybe I'm getting mixed up between "required by the profile form" and "required by the profile API endpoint".